DSCResources/MSFT_xNetBIOS/MSFT_xNetBIOS.psm1

```diff
@@ -0,0 +1,112 @@
+Add-Type -TypeDefinition @'
+   public enum NetBiosSetting
+   {
+      Default,
+      Enable,
+      Disable
+   }
+'@
+
+
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [parameter(Mandatory = $true)]
+        [System.String]
+        $InterfaceAlias,
+
+        [parameter(Mandatory = $true)]
+        [ValidateSet("Default","Enable","Disable")]
+        [System.String]
+        $Setting
+    )
+    try
+    {
+        $NetAdapterConfig = Get-CimInstance -ClassName Win32_NetworkAdapter -Filter "NetConnectionID=`"$InterfaceAlias`"" | Get-CimAssociatedInstance -ResultClassName Win32_NetworkAdapterConfiguration

```

Comment:
Where ever possible you'll want to keep the lines under 100 characters ([DSCGuidelines](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md#each-line-should-have-less-than-100-characters))

ie:
````powershell
$NetAdapterConfig = Get-CimInstance `
    -ClassName Win32_NetworkAdapter
    -Filter "NetConnectionID=`"$InterfaceAlias`"" | 
    Get-CimAssociatedInstance `
        -ResultClassName Win32_NetworkAdapterConfiguration